src/internal/operators/single.ts

```diff
@@ -11,8 +11,34 @@ import { Observer, MonoTypeOperatorFunction, TeardownLogic } from '../types';
  * items, notify of an IllegalArgumentException or NoSuchElementException respectively. If the source Observable
  * emits items but none match the specified predicate then `undefined` is emitted.
  *
+ * <span class="informal">Like {@link first}, but emit with error notification if there is more than one value.</span>
  * ![](single.png)
  *
+ * ## Example
+ * emits 'error'
+ * ```javascript
+ * import { range } from 'rxjs';
+ * import { single } from 'rxjs/operators';
+ *
+ * const numbers = range(1,5).pipe(single());
+ * numbers.subscribe(x => console.log('never get called'), e => console.log('error'));
+ * // result
+ * // 'error'
+ * ```
+ *
+ * emits 'undefined'
+ * ```javascript
```

Comment:
could you change the examples to ts

Reply:
do you mean that change the word 'javascript' to 'typescript',

or use some feature of typescript in the example like interface or type ?

Reply:
What @JWO719 meant was changing

```
```javascript
```

to

```
```ts
```

Reply:
Thank you for the hint !